CMSgov/mgov-design-system
# Header link styles to mds

- [WNMGDS-1040 - Add Link Header styles to MDS](https://jira.cms.gov/browse/WNMGDS-1040) 

## Summary
Medicare has added a style specific to header links. You can learn more about it in the guidelines found here
[Typography Guidelines](https://cms.invisionapp.com/dsm/cms/mgov-design-system/nav/5fa7da3f7044bd0018240dda/folder/60faf7992ed4d43b35eb55d9?mode=preview)


### Added
- Added a header link style that will work automatically when any of the header links adds a <a> tag within the header.
- Added a typography override page to the styles section
- Added a new Header link example to the link html example
- Removed some of the typography changes that correspond more with the WNMGDS-1039 and moved those changes to a new branch

### Changed
- Changed some of the link variables for the offset and text decoration colors

## How to test
You can run the repo locally and navigate to this page locally. [Medicare Design System](https://cmsgov.github.io/mgov-design-system/styles/typography/)

You can see the link example in the link section.
